.gitignore

```diff
@@ -250,23 +250,16 @@ paket-files/
 .idea/
 *.sln.iml
 
-# Visual Studio Code
-.vscode/launch.json
-.vscode/tasks.json
-
-# launchSettings.json
-launchSettings.json
-
 # Node and .NET Core runtime images dockerfiles, that are auto generated
 images/runtime/node/*/Dockerfile
 images/runtime/dotnetcore/*.*/
 images/runtime/python/*.*/
 images/runtime/php/*.*/
 
-# Go packages, if Oryx is added to GOPATH (which it should)
-pkg/
-
 # Go packages from GitHub
-src/github.com/
-src/startupscriptgenerator/vendor/
-src/golang.org/
\ No newline at end of file
+src/startupscriptgenerator/src/github.com/
```

Comment:
I moved all directories which were previously under `startupscriptgenerator` to a sub-directory under it now called `src`. My aim was to keep the structure as per Golang conventions.